src/print.lisp

```diff
@@ -381,4 +589,8 @@
       (t
        (write-string res destination)))))
 
+(defmacro with-input-from-string ((stream string) &body body)
```

Comment:
I don't think this is used anywhere right?  does it work? but it should go in streams.lisp probably.